src/test/kotlin/no/nav/syfo/dialogmotekandidat/api/DialogmotekandidatApiSpek.kt

```diff
@@ -0,0 +1,247 @@
+package no.nav.syfo.dialogmotekandidat.api
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.ktor.client.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.serialization.jackson.*
+import io.ktor.server.testing.*
+import io.mockk.mockk
+import no.nav.syfo.dialogmotekandidat.kafka.DialogmotekandidatEndringProducer
+import no.nav.syfo.oppfolgingstilfelle.toOffsetDatetime
+import no.nav.syfo.testhelper.*
+import no.nav.syfo.testhelper.generator.*
+import no.nav.syfo.util.*
+import org.amshove.kluent.*
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+class DialogmotekandidatApiSpek : Spek({
+    val objectMapper: ObjectMapper = configuredJacksonMapper()
+    val urlKandidatPersonIdent = "$kandidatApiBasePath/$kandidatApiPersonidentPath"
+
+    describe(DialogmotekandidatApiSpek::class.java.simpleName) {
+        val externalMockEnvironment = ExternalMockEnvironment.instance
+
+        val database = externalMockEnvironment.database
+        val dialogmotekandidatEndringProducer = mockk<DialogmotekandidatEndringProducer>()
+
+        fun ApplicationTestBuilder.setupApiAndClient(): HttpClient {
+            application {
+                testApiModule(
+                    externalMockEnvironment = externalMockEnvironment,
+                    dialogmotekandidatEndringProducer = dialogmotekandidatEndringProducer,
+                )
+            }
+            val client = createClient {
+                install(ContentNegotiation) {
+                    jackson { configure() }
+                }
+            }
+            return client
+        }
+
+        beforeEachTest {
+            database.dropData()
+        }
+
+        val validToken = generateJWT(
+            audience = externalMockEnvironment.environment.azure.appClientId,
+            issuer = externalMockEnvironment.wellKnownInternalAzureAD.issuer,
+            navIdent = UserConstants.VEILEDER_IDENT,
+        )
+        describe("Get DialogmoteKandidatDTO for person") {
+            describe("Happy path") {
+                it("returns kandidat=false for person that has never been Kandidat") {
+                    testApplication {
+                        val client = setupApiAndClient()
+                        val response = client.get(urlKandidatPersonIdent) {
+                            bearerAuth(validToken)
+                            header(NAV_PERSONIDENT_HEADER, UserConstants.ARBEIDSTAKER_PERSONIDENTNUMBER.value)
+                        }
+                        response.status shouldBeEqualTo HttpStatusCode.OK
+
+                        val dialogmotekandidatDTO =
```

Comment:
Sånn som man konfigurerer `client` i `setupApiAndClient` så skal det gå an å gjøre:
`val dialogmotekandidatDTO = response.body<DialogmotekandidatDTO>()` i steded for å bruke `objectMapper` rundtomkring.